Header header 
sensor_msgs/Image ImageLeft
sensor_msgs/Image ImageRight
sensor_msgs/Image ImageMask 
sensor_msgs/Image ImageFlo
sensor_msgs/Image ImageFloVis
string ImagePath
string ImageMaskPath 
